ci/testkomodo.sh

```diff
@@ -1,35 +1,52 @@
-copy_test_files () {
-    mkdir $CI_TEST_ROOT/testpath/
-
-    pushd $CI_TEST_ROOT/testpath
-    cp -r $CI_SOURCE_ROOT/tests tests
-    cp -r $CI_SOURCE_ROOT/conftest.py conftest.py
-    ln -s $CI_SOURCE_ROOT/examples
-    ln -s $CI_SOURCE_ROOT/xtgeo-testdata
-    git clone --depth=1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-    popd
+XTG_TEST_RUNPATH=$CI_TEST_ROOT/run
+
+run_tests() {
+    copy_test_files
+    start_tests
+    cleanup
 }
 
-install_package () {
-    pip install ".[dev]"
+copy_test_files () {
+    git clone --depth=1 https://github.com/equinor/xtgeo-testdata $CI_TEST_ROOT/xtgeo-testdata
+    # xtgeo tests look two directories back for xtgeo-testdata
+    mkdir -p $XTG_TEST_RUNPATH
+    cp -r $CI_SOURCE_ROOT/tests $XTG_TEST_RUNPATH
+    ln -s $CI_SOURCE_ROOT/conftest.py $XTG_TEST_RUNPATH/conftest.py
+    ln -s $CI_SOURCE_ROOT/pyproject.toml $XTG_TEST_RUNPATH/pyproject.toml
 }
 
 start_tests () {
-    pushd $CI_TEST_ROOT/testpath
-    pytest -n auto -vv
+    pushd $CI_TEST_ROOT
+    install_and_test
+    run_roxar_tests
     popd
 }
 
-cleanup () {
-    rm -rf $CI_TEST_ROOT/testpath/../xtgeo-testdata
+install_and_test () {
+    install_package
+    run_pytest
 }
 
-run_tests() {
-    copy_test_files
+install_package () {
+    pushd $CI_SOURCE_ROOT
+    pip install ".[dev]"
+    popd
+}
 
-    install_package
+run_pytest () {
+    pushd $XTG_TEST_RUNPATH
+    pytest -n 4 -vv
+    popd
+}
 
-    pushd $CI_TEST_ROOT
-    start_tests
-    cleanup
+run_roxar_tests () {
+    set +e
+    source /project/res/roxapi/aux/roxenvbash latest
```

Comment:
I don't think we should `latest` is this often is a beta version. Rather, it should test a specified matrix of RMS versions

Reply:
Good point, I have added it.